src/components/SurvivorSelection.tsx

```diff
@@ -0,0 +1,40 @@
+import "../styles/survivor.css";
+import { Survivor, UserSelection } from "../utilities/types";
+
+export interface Props {
+  selectedSurvivor: Survivor;
+  survivorList: Survivor[];
+  setSelectedSurvivor: React.Dispatch<React.SetStateAction<Survivor>>;
+  setUserSelection: React.Dispatch<React.SetStateAction<UserSelection>>;
+}
+
+const SurvivorSelection = ({
+  selectedSurvivor,
+  survivorList,
+  setSelectedSurvivor,
+  setUserSelection
+}: Props) => {
+
+  console.log(selectedSurvivor);
+  
+  const handleSelection = (survivor: Survivor) => {
+    setSelectedSurvivor(survivor)
+    setUserSelection((prevUserSelection) => ({...prevUserSelection, id:Date.now(), userSurvivor: survivor}))
+  }
+  
+
+  return (
+    <>
+      <div className="survivor-selection">
+        {survivorList.map((survivor) => (
```

Comment:
continue with the strong typing, even in array functions

```suggestion
        {survivorList.map((survivor: Survivor) => (
```